kubernetes-replicator.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: kubernetes-replicator
+  version: 2.10.2
+  epoch: 0
+  description: Kubernetes controller for synchronizing secrets & config maps across namespaces
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/mittwald/kubernetes-replicator
+      tag: v${{package.version}}
+      expected-commit: 8f4de965a57b974b446fae30048269894fd4ff3d
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -X main.version=${{package.version}}
```

Comment:
I don't see upstream setting this or exposing this variable to set the variable in their main.go at root of the repo so I think we can omit this. 

But this is good to have not a blocker given `go build` with wrong/any ldflags fails silently.

Reply:
All build flags now removed. Thank you